src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html

```diff
@@ -79,66 +64,93 @@
                     </p>
 
                     <ol>
                         <li>
-                          
+                            <p>
                                 From a command prompt, create a new project by entering the following (all on one line):
-                            
+                            </p>
 
-                            <pre class="prettyprint"><code>mvn archetype:generate -DarchetypeGroupId=com.lightbend.lagom -DarchetypeArtifactId=maven-archetype-lagom-java -DarchetypeVersion=1.3.0</code></pre>
+                            <pre class="prettyprint"><code class="language-sh">mvn archetype:generate -DarchetypeGroupId=com.lightbend.lagom -DarchetypeArtifactId=maven-archetype-lagom-java -DarchetypeVersion=@context.currentLagomVersion</code></pre>
                         </li>
-
                         <li>
-                            Maven prompts you for:
-                            <p><ul><li>The number corresponding with the lagom archetype, at this time only one is available, enter the number <code>1</code> to select it.</li>
-                              <li>The Lagom version, press <b>Enter</b> to accept the default.</li>
-                              <li><code>groupId</code> &#8212; becomes the package name, typically something like com.example.com. </li>
-                              <li><code>artifactId</code> &#8212; becomes the top-level folder name, for example, my-first-project.</li>
-                              <li><code>version</code> &#8212; the version for your project, press <b>Enter</b> to accept the default.</li>
-                              <li><code>package</code> &#8212; defaults to the same value as the <code>groupId</code>
-                        </li></ul></p>
-                         <li>Enter <code>Y</code> to accept the values. Maven creates the project and downloads all of the necessary dependencies.</li>
+                            <p>Maven prompts you for:</p>
+                            <ul>
+                                <li>
+                                    <code>groupId</code> &#8212; typically something like com.example.
+                                </li>
+                                <li>
+                                    <code>artifactId</code> &#8212; becomes the top-level folder name, for example,
+                                    my-first-project.
+                                </li>
+                                <li>
+                                    <code>version</code> &#8212; the version for your project, press <b>Enter</b> to
+                                    accept the default.
+                                </li>
+                                <li>
+                                    <code>package</code> &#8212; defaults to the same value as the <code>groupId</code>
+                                </li>
+                            </ul>
```

Comment:
We have to review the lists' styles. This seems to be using a _not great_ padding: 

![screen_shot_2017-03-07_at_14_27_29](https://cloud.githubusercontent.com/assets/762126/23658311/4dd11c0a-0342-11e7-865c-0cd7b5774773.png)

See also #32